src/Symfony/Component/HttpFoundation/Session/Flash/FlashBagInterface.php

```diff
@@ -72,6 +72,8 @@ public function all();
 
     /**
      * Sets all flash messages.
+     * 
+     * @param array $messages
```

Comment:
the missing docblock was intended: we don't add them when they don't provide value over the signature

Reply:
I agree, it's already typed in the parameters of the function